core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -1093,6 +1094,20 @@ wasm_interp_call_func_native(WASMModuleInstance *module_inst,
     if (!ret)
         return;
 
+#if WASM_ENABLE_GC != 0
+    func_type = cur_func->u.func_import->func_type;
+    if (func_type->result_count
+        && wasm_is_type_reftype(func_type->types[cur_func->param_count])) {
+        frame_ref = (uint8 *)prev_frame->csp_boundary
+                    + (unsigned)(prev_frame->sp - prev_frame->lp);
```

Comment:
Had better `(unsigned)(uintptr_t)(prev_frame->sp - prev_frame->lp)`, or compilation warning may be reported?

Reply:
done